src/common/wrap/wrap-events.js

```diff
@@ -79,7 +76,32 @@ export function wrapEvents(sharedEE) {
 
   return ee
 }
+/**
+ * Find the base prototype of 'object' that has its own "addEventListener" property, and run some function on it.
+ * @param {Object} object - the initial object to traverse prototype chain on
+ * @param {Function} cb - the function to run on the ancestral object once found, accepts an object as a arg
+ * @param {Array} rest - [optional] any additional arguments to pass to the cb
+ */
+function findEventListenerProtoAndCb(object, cb, ...rest) {
+  let step = object;
+  // eslint-disable-next-line
```

Comment:
Should not be adding new eslint disable line statements.

Reply:
This had existed; will disabling make lint complain?